configurations/dps-validation-service.conf

```diff
@@ -0,0 +1,14 @@
+###############################################################################
+##
+##            DPS Validation Service Specific Environment Variables
+##            Author: @akroon3r
+##
+###############################################################################
+
+SOURCE_CONTEXT_DIR="/src/dpsvalidationservice"
+BUILD_CONFIG_TEMPLATE="dps-validation-service.bc.yaml"
+DEPLOY_CONFIG_TEMPLATE="dps-validation-service.dc.yaml"
+APP_NAME="dps-validation-service"
+BUILD_REQUIRED="false"
+# TODO Change value for DFCMS_BASE_PATH
+DFCMS_BASE_PATH="${DFCMS_BASE_PATH}"
```

Comment:
this should be set on openshift env, there should be 3 values:

- DFCMS_BASE_PATH
- DFCMS_USERNAME
- DFCMS_PASSWORD

Reply:
added blank values to the config